Sample Project/SimpleLineChart/ViewController.m

```diff
@@ -240,7 +240,7 @@ - (void)lineGraphDidFinishLoading:(BEMSimpleLineGraphView *)graph {
     // Use this method for tasks after the graph has finished drawing
 } */
 
-- (NSString *)popUpSuffixForlineGraph:(BEMSimpleLineGraphView *)graph {
+- (NSString *)popUpSuffixForlineGraph:(BEMSimpleLineGraphView *)graph index:(NSUInteger)index{
     return @" people";
 }
 
```

Comment:
The commented out method on line 247 also needs to be updated with the new parameter `index`.